packages/web/src/components/card/Card.tsx

```diff
@@ -49,10 +56,13 @@ export type CardContentProps = FlexProps
 
 export const CardContent = (props: CardContentProps) => {
   const { size } = useContext(CardContext)
+  const { spacing } = useTheme()
+  const cardWidth = cardSizes[size]
 
   return (
     <Flex
       direction='column'
+      w={cardWidth ? cardWidth - spacing.s * 2 : undefined}
```

Comment:
This feels a bit awkward. Is there a reason standard padding and width=100% doesn't handle this properly?

Reply:
yeah let me double check